tests/pre_merge/tools/test_gradio_entrypoint.py

```diff
@@ -36,6 +37,8 @@ def test_torch_inference(self, get_functions, project_path, get_config, transfor
         """Test gradio_inference.py"""
         parser, inferencer = get_functions
         model = get_model(get_config("padim"))
+        model.image_threshold = F1AdaptiveThreshold()
+        model.pixel_threshold = F1AdaptiveThreshold()
```

Comment:
why do we need to add them now? Is it because we do not initialize `Engine` for these tests?

Reply:
is it not possible to use `dummy_lightning_model` here?

Reply:
I've removed initializing these from the base module as we attach the threshold based on the class passed to the Engine. I was thinking of using dummy lightning module but it will involve copying it in this folder. I was thinking of postpoing this till we refactor the tests. We have almost the same model somewhere. Maybe we can have a common conftest.py in the root of the tests.